runtime/src/memo.rs

```diff
@@ -0,0 +1,29 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use srml_support::{StorageMap, dispatch::Result, decl_module, decl_storage, ensure};
+use srml_support::traits::{Currency};
+use runtime_primitives::traits::{Zero};
+use system::{self, ensure_signed};
+use rstd::prelude::*;
+use crate::governance::GovernanceCurrency;
+
+pub trait Trait: system::Trait + GovernanceCurrency {}
+
+decl_storage! {
+    trait Store for Module<T: Trait> as Memo {
+        Memo get(memo) : map T::AccountId => Vec<u8>;
```

Comment:
what if this? it will allow to save different memos uner different keys per account.
`Memo get(memo) : map (T::AccountId, Vec<u8>) => Vec<u8>;`

Reply:
That's a good suggestion and it probably needs some more thought. What predefined keys do we want to have. Don't want to add too much here, as it might have some duplicate features which @bedeho might be working on for the membership module. I suggest we add it as an enhancement in a future upgrade, as a new storage value, `Field`, if we decide to keep this module at all.